src/components/DateRangePicker.tsx

```diff
@@ -11,20 +11,33 @@ import { globalDateCache } from '../helpers/date-cache';
 import { useResizeDetector } from 'react-resize-detector';
 
 interface Props {
+  submission?: boolean;
   dateRangeSelector: DateRangeSelector;
   setDateRangeSelector?: React.Dispatch<React.SetStateAction<DateRangeSelector>>;
+  setSubmissionDateRangeSelector?: React.Dispatch<React.SetStateAction<DateRangeSelector>>;
+  setSpecialSubmissionDateRaw?: React.Dispatch<React.SetStateAction<string | null>>;
+  specialSubmissionDateRaw?: string | null;
```

Comment:
Why do we need to distinguish between the submission date range selector and the regular date range selector here? Why does it make a difference for the DateRangePicker? (I understand the need for the `submission` field but not exactly why we need `setSubmissionDateRangeSelector, setSpecialSubmissionDateRaw, and specialSubmissionDateRaw`.